python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1708,12 +1710,18 @@ def findArtifacts(self, templateCoadd, tempExpRefList, imageScalerList):
             if mi is not None:
                 chiIm = self._makeChiIm(mi)
                 chiOneMap = self._snrToBinaryIm(chiIm)
-                epochCountImage += chiOneMap
                 outliers = afwImage.makeMaskFromArray(chiOneMap.array.astype(afwImage.MaskPixel))
                 outliers.setXY0(mi.getXY0())
                 spanSetList = afwGeom.SpanSet.fromMask(outliers).split()
+                dilatedSpanSetList = [s.dilated(self.config.growMaskBy,
+                                                afwGeom.Stencil.CIRCLE).clippedTo(coaddBBox)
+                                      for s in spanSetList]
+                slateIm.set(0)
+                for spans in dilatedSpanSetList:
+                    spans.setImage(slateIm, 1, doClip=True)
+                epochCountImage += slateIm
             else:
-                chiIm = afwImage.ImageF(templateCoadd.getBBox(), numpy.nan)
+                chiIm = afwImage.ImageF(coaddBBox, numpy.nan)
                 spanSetList = []
```

Comment:
I think this needs to be renamed to dilatedSpanSetList, otherwise there is a chance below that spanSetArtifactList.append will fail, as spanSetList will not exist.